### DISCUSSION TITLE
Set SVG stylesheet / currentColor

### DISCUSSION THREAD

**Discussion by szechyjs** [libvips/libvips#4333] [Ideas]
Many SVGs use style information from their parent/host view when rendering. A good example of this is `currentColor` for fill/stroke. By default librsvg uses black `#000000`, however it would be nice to be able to set a stylesheet so we can override `color` to something else. I believe an interface is exposed for this: `rsvg_handle_set_stylesheet`.

Links:
https://gnome.pages.gitlab.gnome.org/librsvg/internals/rsvg/property_defs/struct.Color.html
https://gnome.pages.gitlab.gnome.org/librsvg/Rsvg-2.0/method.Handle.set_stylesheet.html

**Comment by jcupitt** [comment]
Hi @szechyjs, sure, this sounds like a good idea. Let's make this an enhancement.

**Reply by jcupitt** [reply]
https://github.com/libvips/libvips/issues/4334
